WS2012R2/lisa/setupscripts/NET_ADD_Switch.ps1

```diff
@@ -45,38 +45,12 @@
    All setup and cleanup scripts must return a boolean ($true or $false)
    to indicate if the script completed successfully or not.
 
-   .Parameter vmName
-	Name of the VM to add NIC to .
-
-	.Parameter hvServer
```

Comment:
Please let's keep the hvServer param and any code should be tested on a remote hvServer.
This is to cover cases with the nano, server core or Hyper-V Server.